int64 key
float32[3] xyz
float32[3] rpy
float32[6] sigmas